juriscraper/opinions/united_states/state/nh.py

```diff
@@ -1,127 +1,150 @@
-# Scraper for New Hampshire Supreme Court
-# CourtID: nh
-# Court Short Name: NH
-# Court Contact: webmaster@courts.state.nh.us
-# Author: Andrei Chelaru
-# Reviewer: mlr
-# History:
-# - 2014-06-27: Created
-# - 2014-10-17: Updated by mlr to fix regex error.
-# - 2015-06-04: Updated by bwc so regex catches comma, period, or whitespaces
-#   as separator. Simplified by mlr to make regexes more semantic.
-# - 2016-02-20: Updated by arderyp to handle strange format where multiple
-#   case names and docket numbers appear in anchor text for a single case
-#   pdf link. Multiple case names are concatenated, and docket numbers are
-#   concatenated with ',' delimiter
+"""
+Scraper for New Hampshire Supreme Court
+CourtID: nh
+Court Short Name: NH
+Court Contact: webmaster@courts.state.nh.us
+Author: Andrei Chelaru
+Reviewer: mlr
+History:
+    - 2014-06-27: Created
+    - 2014-10-17: Updated by mlr to fix regex error.
+    - 2015-06-04: Updated by bwc so regex catches comma, period, or
+    whitespaces as separator. Simplified by mlr to make regexes more semantic.
+    - 2016-02-20: Updated by arderyp to handle strange format where multiple
+    case names and docket numbers appear in anchor text for a single case pdf
+    link. Multiple case names are concatenated, and docket numbers are
+    concatenated with ',' delimiter
+    - 2021-12-29: Updated for new web site, by flooie and satsuki-chan
+Notes:
+    To filer documents per year, the API expects the key number for the year
+    in parameter:
+    - 'tag': for years 2019 to date
+        * 2021: tag=1206    * 2020: tag=1366    * 2019: tag=1416
+    - 'subcategory': for years 2009 to 2018
+        * 2018: subcategory=1601    * 2017: subcategory=1596
+        * 2016: subcategory=1591    * 2015: subcategory=1586
+        * 2014: subcategory=1581    * 2013: subcategory=1576
+        * 2012: subcategory=1571    * 2011: subcategory=1566
+        * 2010: subcategory=1561    * 2009: subcategory=1556
+    Only one parameter can have a key number; using both with a value in the
+    same request (even with a valid key) returns zero documents.
+    Using none, returns all documents (regardless of the publication year)
+    found with the other parameters values.
+    For this scraper, filtering documents per year is not necesary.
+    Examples:
+        - https://www.courts.nh.gov/content/api/documents?text=&category=+undefined&purpose=1331+undefined&tag=+&subcategory=1556&sort=field_date_posted|desc&page=3&size=10
+        - https://www.courts.nh.gov/content/api/documents?text=&category=+undefined&purpose=1331+undefined&tag=1366+&subcategory=&sort=field_date_posted|desc&page=1&size=25
+"""
 
 import re
-from datetime import date
+from typing import Tuple
```

Comment:
If you remove that method you can delete this import as well.

Reply:
Updated the code for the method, so the import remains.